src/applications/personalization/rated-disabilities/components/RatedDisabilityListItem.jsx

```diff
@@ -7,55 +7,35 @@ const RatedDisabilityListItem = ({ ratedDisability }) => {
     ratingPercentage,
     name,
     effectiveDate,
-    relatedTo,
   } = ratedDisability;
   return (
-    <div className="vads-l-col--12 vads-u-background-color--gray-lightest vads-u-margin-top--0 vads-u-margin-bottom--2 vads-u-padding-top--1 vads-u-padding-bottom--2 vads-u-padding-x--2">
-      <div className="vads-l-row">
-        <div className="vads-l-col--9">
-          <p className="vads-u-font-weight--bold vads-u-margin-top--0p25 vads-u-margin-bottom--0 vads-u-margin-x--0 vads-u-font-size--base">
-            {name}
-          </p>
-        </div>
-        {typeof ratingPercentage === 'number' ? (
-          <div className="vads-l-col--3">
-            <p className="vads-u-font-weight--bold vads-u-font-size--lg vads-u-text-align--right vads-u-margin--0">
-              {ratingPercentage}%
-            </p>
-          </div>
-        ) : null}
-      </div>
-      <div className="vads-l-row vads-u-margin-y--0p5">
-        <div className="vads-l-col--12 vads-u-display--flex vads-u-flex-direction--column small-screen:vads-u-flex-direction--row">
-          <p className="vads-u-margin--0">
-            {decisionText === 'Service Connected' ? (
-              <i
-                className="fas fa-medal vads-u-margin-right--0p5"
-                aria-hidden="true"
-                role="img"
-              />
-            ) : null}
-            <span className="vads-u-font-weight--bold">{decisionText}</span>{' '}
-          </p>
-          {relatedTo ? (
-            <>
-              <p className="vads-u-display--none small-screen:vads-u-display--inline vads-u-margin-x--0p5 vads-u-margin-y--0">
-                {' '}
-                |{' '}
-              </p>
-              <p className="vads-u-margin--0">Related To: {relatedTo}</p>
-            </>
-          ) : null}
-        </div>
-      </div>
+    <dl className="vads-u-display--block vads-l-col--12 vads-u-margin--0  vads-u-background-color--gray-lightest vads-u-margin-top--0 vads-u-margin-bottom--2 vads-u-padding-top--1 vads-u-padding-bottom--2 vads-u-padding-x--2">
+      {typeof ratingPercentage === 'number' ? (
+        <dt className="vads-u-display--inline-block vads-u-font-size--h3 vads-u-font-weight--bold vads-u-margin--0 vads-u-border-color--gray-light vads-u-border-bottom--1px">
+          {ratingPercentage}%
```

Comment:
I haven't used description lists much, so if there's a better way to build this let me know.